modules/distributor/forwarder_test.go

```diff
@@ -91,11 +91,16 @@ func TestForwarder_pushesQueued(t *testing.T) {
 		assert.Equal(t, 0, len(f.queueManagers[tenantID].reqChan))
 	}()
 
+	wg := sync.WaitGroup{}
+	wg.Add(11)
```

Comment:
move this to line 98 and change it to `wg.Add(1)`. that way if someone changes the for loop length the waitgroup still works.